src/View/Helper/FormHelper.php

```diff
@@ -2481,7 +2481,7 @@ public function context(?ContextInterface $context = null): ContextInterface
      */
     protected function _getContext(mixed $data = []): ContextInterface
```

Comment:
Looks like this must be an array.

Reply:
Is that a separate pr maybe?

Reply:
I would like to get this merged, so I can then then tackle other topics.